src/main/java/duke/command/Parser.java

```diff
@@ -0,0 +1,77 @@
+package duke.command;
+
+import duke.dukeexception.DukeException;
+import duke.dukeexception.NoCommandException;
+import duke.dukeexception.NoDescriptionException;
+
+/**
+ * Types of user commands.
+ * User has to type these commands in each line of input
+ * to command the bot.
+ */
+enum CommandType {
+    LIST,
+    DONE,
+    DELETE,
+    BYE,
+    TODO,
+    DEADLINE,
+    EVENT,
+    FIND
+}
```

Comment:
Perhaps you can create a separate java file for this enum class since it is used by other classes too (DoneCommand, AddCommand, etc)